### PR TITLE
Added logging to have more details.

### PR DESCRIPTION
**Pull request by DhobaleVishal**
Signed-off-by: Vishal Dhobale <vishal.dhobale@seagate.com>

# Problem Statement
- Added code to have more details about object size and operations.

# Design
-  For Bug, Describe the fix here.
-  For new test addition post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed
-  [ ] Attach test execution logs

# Review Checklist 
  Checklist for Author
-  [x] JIRA number/GitHub Issue added to PR
-  [x] PR is self reviewed
-  [x] Jira and state/status is updated and JIRA is updated with PR link
-  [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe

### CODE REVIEW

File: scripts/s3/s3api/bucket_operations.py [hunk 1]
```diff
@@ -57,29 +58,30 @@ def __init__(self, access_key: str, secret_key: str, endpoint_url: str, test_id:
...
         while True:
-            LOGGER.info("Iteration %s is started...", self.iteration)
+            LOGGER.info("Iteration %s is started for %s...", self.iteration, self.test_id)
             try:
-                if not isinstance(self.object_size, dict):
-                    file_size = self.object_size
-                else:
-                    file_size = random.randrange(self.object_size["start"], self.object_size["end"])
+                file_size = random.randrange(self.object_size["start"], self.object_size["end"])\
+                    if isinstance(self.object_size, dict) else self.object_size
```
Comment (ketanarlulkar) [inline scripts/s3/s3api/bucket_operations.py]: Why make it one line `if else` if it's not fitting in one line. Takes away all the readability.

File: scripts/s3/s3api/copy_object.py [hunk 2]
```diff
@@ -72,23 +70,31 @@ async def execute_copy_object_workload(self):
...
-                    file_size = self.object_size
-                else:
-                    file_size = random.randrange(self.object_size["start"], self.object_size["end"])
-                with open(object1, 'wb') as fout:
-                    fout.write(os.urandom(file_size))
+                file_size = random.randrange(
+                    self.object_size["start"],
+                    self.object_size["end"]) if isinstance(
+                    self.object_size,
+                    dict) else self.object_size
```
Comment (ketanarlulkar) [inline scripts/s3/s3api/copy_object.py]: Same comment as above. Previous code was clean and reasonable to read and understand.

File: scripts/s3/s3api/bucket_operations.py (hunk 1 above)
Comment (DhobaleVishal) [reply scripts/s3/s3api/bucket_operations.py]: This is readable and not a complex operations so anyone can read it easily.

File: scripts/s3/s3api/copy_object.py (hunk 2 above)
Comment (DhobaleVishal) [reply scripts/s3/s3api/copy_object.py]: Addressed this in previous comment.

File: scripts/s3/s3api/bucket_operations.py (hunk 1 above)
Comment (ketanarlulkar) [reply scripts/s3/s3api/bucket_operations.py]: But I could not understand the reason to change it. IMHO, this is readable but previous code was more readable than current code.

Comment (DhobaleVishal) [reply scripts/s3/s3api/bucket_operations.py]: Okay let me correct all script. @ketanarlulkar Do you want all conditions to be like this?

Comment (ketanarlulkar) [reply scripts/s3/s3api/bucket_operations.py]: Just trying to understand the reason. If you think it adds value, you can keep it.

**Review by ketanarlulkar — APPROVED** [review]

**Review by VikasKumarC — APPROVED** [review]

**Pull request merged**